README.md

```diff
@@ -29,6 +29,7 @@ Table of Contents
          * [Session and Transaction Delay Specification](#session-and-transaction-delay-specification)
       * [Traffic Verification Specification](#traffic-verification-specification)
          * [Field Verification](#field-verification)
+            * [meta headers](#meta-headers)
```

Comment:
It's good to add this section under the `Traffic Verification Specification` section, but let's also add information about trailer replay in the `Traffic Replay Specification` section.